packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -141,29 +172,51 @@ export default function LabeledField(props: Props) {
     }
 
     function maybeRenderError(): React.ReactNode | null | undefined {
-        if (!error) {
-            return null;
-        }
-
         return (
             <React.Fragment>
                 <Strut size={spacing.small_12} />
-                <LabelSmall
-                    style={light ? styles.lightError : styles.error}
-                    role="alert"
-                    id={errorId}
-                    testId={testId && `${testId}-error`}
+                <View
+                    style={styles.errorSection}
+                    id={errorSectionId}
+                    testId={testId && `${testId}-error-section`}
```

Comment:
**suggestion:** This made me realize that we shouldn't use `testId=${testId}-error` in the error label (below) and instead just use a single testId for the error itself. This would also simplify testing the error state.

 ```suggestion
                    testId={testId && `${testId}-error`}
```